### PR TITLE
Selects the selected vhost as default

### PR DESCRIPTION
**Pull request by kickster97**
When creating a queue, the default selected vhost is now the overall chosen vhost (right top corner of page), instead of `/`, which some user may not have access to.

### CODE REVIEW

**Review by carlhoerberg — CHANGES_REQUESTED** [review]

File: static/queues.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html lang="en">
+
```
[the remark this answers is not in the document]
Comment (kickster97) [reply static/queues.html:3]: I included it bc my linter corrected the file, but maybe i could do a focused lint job instead some time and comb through everything.

[the remark this answers is not in the document]
Comment (carlhoerberg) [reply static/queues.html:3]: Correct

File: static/js/vhosts.js [hunk 2]
```diff
@@ -27,15 +27,17 @@ function addVhostOptions (formId) {
...
-      err.textContent = "Error fetching data: Please try to refresh the page!"
+      err.textContent = 'Error fetching data: Please try to refresh the page!'
       select.parentElement.insertAdjacentElement('beforebegin',err)
       return
     }
 
+    const selectedVhost = window.sessionStorage.getItem('vhost')
     for (let i = 0; i < vhosts.length; i++) {
       const opt = document.createElement('option')
       opt.text = vhosts[i].name
```
Comment (carlhoerberg) [inline static/js/vhosts.js:38]:
```suggestion
```

Comment (kickster97) [reply static/js/vhosts.js:38]: without the text setting, the content of the option tag will be empty

Comment (kickster97) [reply static/js/vhosts.js:38]: I could use the label attribute instead if that is preferred? or is there another way?

**Review by carlhoerberg — COMMENTED** [review]
Revert the changes in queue.html, and then you can squash merge.

File: static/js/vhosts.js (hunk 2 above)
Comment (carlhoerberg) [reply static/js/vhosts.js]: Ok, docs says that value would be used if no label was set.

**Review by carlhoerberg — APPROVED** [review]

**Pull request merged**